test/test_issue801.py

```diff
@@ -13,7 +13,8 @@ def test_issue_801(self):
         g.bind('', example)
         node = BNode()
         g.add((node, example['first%20name'], Literal('John')))
-        print(g.serialize(format="turtle").decode())
+        self.assertEqual(g.serialize(format="turtle").decode().split("\n")[-3] == '[] :first%20name "John" .', True)
+        print (g.serialize(format="turtle").decode())
```

Comment:
This is going to make test run output noisy. I don't see other tests in rdflib printing output, so do you need this?

Reply:
@darrengarvey I have applied the requested changes